apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,31 @@
+import { IDetailQueryResult, ITranscribedAudioViewModel } from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import './transcribed-audio-detail.full-view.presenter.css';
+
+export const TranscribedAudioDetailFullViewPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const runTime = Math.round(transcribedAudioItem.length / 1000);
```

Comment:
Declare a constant
```ts
const MILLISECONDS_PER_SECOND = 1000
```
for clarity (no magic numbers).

Reply:
Done